packages/flutter/lib/src/widgets/scrollable.dart

```diff
@@ -319,9 +321,14 @@ class ScrollableState<T extends Scrollable> extends State<T> {
   }
   ExtentScrollBehavior _scrollBehavior;
 
-  /// Subclasses should override this function to create the [ScrollBehavior]
-  /// they desire.
-  ExtentScrollBehavior createScrollBehavior() => new OverscrollWhenScrollableBehavior();
+  /// Use the value returned by [ScrollConfiguration.createScrollBehavior].
+  /// If this widget doesn't have a ScrollConfiguration ancestor,
+  /// or its createScrollBehavior callback is null, then return a new instance
+  /// of [OverscrollWhenScrollableBehavior].
+  ExtentScrollBehavior createScrollBehavior() {
+    final ValueGetter<ExtentScrollBehavior> create = ScrollConfiguration.of(context)?.createScrollBehavior;
+    return create != null ? create() : new OverscrollWhenScrollableBehavior();
```

Comment:
It seems like it would be better to put this default in `ScrollConfiguration.of`